packages/eventcatalog/components/SchemaViewer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react';
+
+import { JsonSchemaViewer } from '@stoplight/json-schema-viewer';
+import { injectStyles } from '@stoplight/mosaic';
+import styles from './SchemaViewer.module.css';
+
+// Inject Stoplight styles
+injectStyles();
+
+type Props = {
+  schema: any;
+  renderRootTreeLines?: boolean;
+  hideExamples?: boolean;
+  defaultExpandedDepth?: number;
+};
+
+function SchemaViewer({ schema, renderRootTreeLines = false, hideExamples = false, defaultExpandedDepth = 1 }: Props) {
+  const JsonSchema = useMemo(() => JSON.parse(schema as string), [schema]);
+
+  return (
+    <JsonSchemaViewer
+      schema={JsonSchema}
+      emptyText="No schema defined"
+      defaultExpandedDepth={defaultExpandedDepth}
+      // viewMode='standalone'
+      renderRootTreeLines={Boolean(renderRootTreeLines)}
```

Comment:
I think we are OK not to cast as boolean, as the types should help here and think it's safe to assume boolean coming into props?
```suggestion
      renderRootTreeLines={renderRootTreeLines}
```